jsonnet/components/node-exporter.libsonnet

```diff
@@ -3,6 +3,31 @@ local textfileVolumeName = 'node-exporter-textfile';
 local tlsVolumeName = 'node-exporter-tls';
 local wtmpPath = '/var/log/wtmp';
 local wtmpVolumeName = 'node-exporter-wtmp';
+local configDir = '/var/node_exporter/config';
+local configVolumeName = 'node-exporter-config';
+local acceleratorsConfigFileName = 'accelerators-config.yaml';
+local acceleratorsConfigMapName = 'accelerators-configmap';
+local acceleratorsConfigData = |||
```

Comment:
you can use `std.manifestYamlDoc()` to convert a valid jsonnet object to its YAML representation. It will avoid any indentation issue.
https://jsonnet.org/ref/stdlib.html

Reply:
Fixed